lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -255,17 +255,27 @@ def log.trace?
 
     end
 
-    def get_metadata_for_record(match_data, cache_key, create_time, batch_miss_cache)
+    def get_metadata_for_record(match_data, container_id, create_time, batch_miss_cache)
       namespace_name = match_data['namespace']
       pod_name = match_data['pod_name']
+      container_name = match_data['container_name']
       metadata = {
-        'container_name' => match_data['container_name'],
-        'namespace_name' => namespace_name,
-        'pod_name'       => pod_name
+        'container_name'  => container_name,
+        'namespace_name'  => namespace_name,
+        'pod_name'        => pod_name,
+        'container_image' => 'unknown'
```

Comment:
I would consider not defaulting this value given the amount of data we transfer for each message is already large.  Couldn't we consider we dont know what it is if its not there.